driver-examples/tycho/.gitignore

```diff
@@ -0,0 +1,2 @@
+!.classpath
```

Comment:
I am unignoring these to make it simpler for users to import these projects in Eclipse.